lib/internal/loader/Loader.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const URL = require('url').URL;
+
+const ModuleWrap = process.binding('module_wrap').ModuleWrap;
```

Comment:
For consistency with the `const { Error } ...` line below, perhaps make these...

```js
const { URL } = require('url');
const { ModuleWrap } = process.binding('module_wrap');
```